ci/release/update-version.sh

```diff
@@ -56,3 +56,8 @@ for FILE in .github/workflows/*.yaml; do
   sed_runner "/shared-workflows/ s/@.*/@branch-${NEXT_SHORT_TAG}/g" "${FILE}"
 done
 sed_runner "s/RAPIDS_VERSION_NUMBER=\".*/RAPIDS_VERSION_NUMBER=\"${NEXT_SHORT_TAG}\"/g" ci/build_docs.sh
+
+# Docs referencing source code
+for FILE in docs/source/*.rst; do
+    sed_runner "s/branch-${CURRENT_SHORT_TAG}\//branch-${NEXT_SHORT_TAG}\//g" "${FILE}"
```

Comment:
It's better not to use the current tag just in case things somehow deviate.

This should work:
```suggestion
    sed_runner "s|/branch-[^/]*/|/branch-${NEXT_SHORT_TAG}/|g" "${FILE}"
```

Reply:
Ah yes, that's a much better solution, thanks Ray!